bait-files/src/main/webapp/submission.js

```diff
@@ -0,0 +1,9 @@
+/** Do not allow spaces in input */
+function checkSpace(event)
+{
+   if(event.which ==32)
```

Comment:
event.which is deprecated.  You can use event.key instead, which will actually return the actual space character, and then you can just test for that instead of having to use the number 32, so that's nice!

Also, use the triple equals whenever possible in javascript =D

if (event.key === ' ')